src/cf-layout.less

```diff
@@ -1605,6 +1605,121 @@
     }
 }
 
+/* topdoc
+  name: Featured content module
+  family: cf-layout
+  patterns:
+    - name: .featured-content-module
+      markup:
+        <section class="block
+                        block__border
+                        block__flush
+                        featured-content-module">
+          <div class="featured-content-module_text">
+            <p class=h4>
+              <span class="featured-content-module_icon cf-icon cf-icon-date">
+              </span>
+              Featured event
+            </p>
+            <h2>Consumer Advisory Board meeting</h2>
+            <p>Oct 22, 2015 @ 10:00AM EDT <br/>Washington, DC</p>
+            <a class="jump-link jump-link__underline">
+              <span class="jump-link_text">Read more about the event</span>
+            </a>
+          </div>
+          <div class="featured-content-module_visual">
+            <img class="featured-content-module_img"
+                 src="http://placekitten.com/g/540/270">
+          </div>
+        </section>
+      notes:
+        - "Featured content module, like a hero, consists of
+           headline/text/optional call to action along with a visual. It is
+           intended to be used in a main content column next to a sidebar."
+        - "Text is full width & displayed above the visual in the
+           default/mobile view. At larger screen sizes, the text occupies a
+           fixed portion of the screen (equal to the width of 5 of 12 columns
+           at 701px & 3 of 12 columns at 901px for desktop). The
+           visual occupies the remaining space. The visual should be 640x360
+           (16x9 ratio) and resize to fit the height of the FCM with a static
+           width and is center-cropped when it becomes too wide for the
+           available space."
+      codenotes:
+        - .featured-content-module
+  tags:
+    - cf-layout
+*/
+
+.featured-content-module {
+  @fcm-bp-max: 700px;
+  @fcm-bp-min: 701px;
```

Comment:
The hero [includes it where it's used](https://github.com/cfpb/cf-layout/blob/gh-pages/src/cf-layout.less#L1647), but outside the selector. At a minimum this should be updated to match that.